reference-architecture/azure-ansible/testcases/test_pv.sh

```diff
@@ -0,0 +1,8 @@
+oc new-project demo
```

Comment:
Do you want to try to migrate to the common validation playbooks here? Less work to maintain and closer to the ansible migration